many-to-many translation in french

akram opened the pull request:
many-to-many translation in french in the context of database cardinality has to be translated by plusieurs à plusieurs

akram committed:
many-to-many translation in french in the context of database cardinality

many-to-many translation in french in the context of database cardinality has to be translated by plusieurs à plusieurs

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/liferay/liferay-portal?pullRequest=5775) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/liferay/liferay-portal?pullRequest=5775) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/liferay/liferay-portal?pullRequest=5775) it.</sub>

akram committed:
many-to-many translation in french

many-to-many translation in french in the context of database cardinality has to be translated by plusieurs à plusieurs
